plugins/irma-web/dom-manipulations.js

```diff
@@ -172,6 +173,14 @@ module.exports = class DOMManipulations {
     `;
   }
 
+  _stateAborted() {
+    return `
+      <!-- State: Error -->
+      <div class="irma-web-forbidden-animation"></div>
+      <p>${this._translations.error}</p>
+    `;
+  }
```

Comment:
Dit lijkt me niet de juiste state om te tonen. Volgens mij verdient dit een eigen tekst en visualisatie :) Maar voor nu misschien gewoon helemaal geen visuele state? Immers, het idee is dat de flow "van buitenaf" wordt geannuleerd. Dan is het ook aan "buitenaf" om dit visueel mooi op te ruimen (door de popups te sluiten, naar een andere pagina te navigeren, whatever)

Reply:
Ja daar heb ik nog wel even over zitten denken, maar het ding is natuurlijk dat een gebruiker het ook helemaal nooit zou moeten kunnen zien. Zolang iets nog zichtbaar is, zou een `Aborted` state eigenlijk niet gebruikt mogen worden. Maar goed als ik het er niet neerzet, gaat het errors gooien met dat er geen rendering is voor die state.

Reply:
Het enige waar het nuttig voor is, is om de previous state nog even een x tijd zichtbaar te houden in de animatie van het verdwijnen van de pop-up bijvoorbeeld. Dat heb ik in de `handle` pull request toegevoegd.

Reply:
Ja, precies. Je wilt eigenlijk helemaal niet dat er visueel iets gebeurt, want dat gaat alleen maar in de weg zitten voor de popup close animatie :)